feat(patterns): a11y - standardize focus states

tiloyi opened the pull request:
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

Standardise the focus state

GitHub issue number or Jira issue URL: [MZC-6](https://jira.adeo.com/browse/MZC-6)

## Other information

caqueste reviewed (changes requested):
Great work @tiloyi 💪

Just a few changes.
- Quantity selector: Can we always have the focus on the foreground?
- Tags: Can we remove the white outline? It's visible with a dark background.
- I forgot the tooltip triggered by an icon. The board is now updated ;)

tiloyi commented:
> Great work @tiloyi 💪
> 
> Just a few changes.
> 
> * Quantity selector: Can we always have the focus on the foreground?
> * Tags: Can we remove the white outline? It's visible with a dark background.
> * I forgot the tooltip triggered by an icon. The board is now updated ;)

@caqueste I have fixed the following points : 
- Quantity selector: Can we always have the focus on the foreground?
- Tags: Can we remove the white outline? It's visible with a dark background.

For the remaining point, I'll let you keep me informed.
